complex.c

```diff
@@ -2506,6 +2506,80 @@ float_arg(VALUE self)
  * - Separately, with methods Complex#real and Complex#imag.
  * - Together, with method Complex#rect.
  *
+ * == What's Here
+ *
+ * First, what's elsewhere:
+ *
+ * - \Class \Complex inherits (directly or indirectly)
+ *   from classes {Numeric}[rdoc-ref:Numeric@What-27s+Here]
+ *   and {Object}[rdoc-ref:Object@What-27s+Here].
+ * - Includes (directly or indirectly)
+ *   modules {Comparable}[rdoc-ref:Comparable@What-27s+Here]
+ *   and {Kernel}[rdoc-ref:Kernel@What-27s+Here].
+ *
+ * Here, class \Complex has methods for:
+ *
+ * - Creating \Complex objects:
+ *
+ *   - ::polar: Returns a new \Complex object based on given polar coordinates.
+ *   - ::rect (and its alias ::rectangular):
+ *     Returns a new \Complex object based on given rectangular coordinates.
+ *
+ * - Querying:
+ *
+ *   - #abs (and its alias #magnitude): Returns the absolute value for +self+.
+ *   - #arg (and its aliases #angle and #phase):
+ *     Returns the argument (angle) for +self+ in radians.
+ *   - #denominator: Returns the denominator of +self+.
+ *   - #finite?: Returns whether both +self.real+ and +self.image+ are finite.
+ *   - #hash: Returns the integer hash value for +self+.
+ *   - #imag (and its alias #imaginary): Returns the imaginary value for +self+.
+ *   - #infinite?: Returns whether +self.real+ or +self.image+ is infinite.
+ *   - #numerator: Returns the numerator of +self+.
+ *   - #polar: Returns the array <tt>[self.abs, self.arg]</tt>.
+ *   - #inspect: Returns a string representation of +self+.
+ *   - #real: Returns the real value for +self+.
+ *   - #real?: Returns +false+; for compatibility with Numeric#real?.
+ *   - #rect (and its alias #rectangular):
+ *     Returns the array <tt>[self.real, self.imag]</tt>.
+ *
+ * - Comparing:
+ *
+ *   - {#<=>}[Complex.html#method-i-3C-3D-3E]:
+ *     Returns whether +self+ is less than, equal to, or greater than the given argument.
+ *   - {#==}[Complex.html#method-i-3D-3D]:
+ *     Returns whether +self+ is equal to the given argument.
```

Comment:
This generates correctly for me, so I don't think we need to manually link to `#<=>` and `#==`.

```suggestion
 *   - #<=>: Returns whether +self+ is less than, equal to, or greater than the given argument.
 *   - #==: Returns whether +self+ is equal to the given argument.
```